Automattic/jetpack
# Jetpack Scan Visibility i1: Re-add jetpack scan submenu to be shown for atomic sites only

Fixes #

## Proposed changes:

* Jetpack menus now has a `Scan` submenu that's only visible on atomic sites.
![image](https://github.com/Automattic/jetpack/assets/47489215/937bda4e-fb09-4f60-bf92-9fd6fa4288df)
 

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test WoA using a dev blog
  * Install [Jetpack Beta Tester Plugin](https://jetpack.com/download-jetpack-beta/)
  * Configure beta plugin `/wp-admin/admin.php?page=jetpack-beta`
  * Configure Jetpack to use the `add/scan-submenu-for-atomic` feature branch
  * Verify submenu is present and shows scan history. Test with https://github.com/Automattic/wp-calypso/pull/90910

* Test on Simple sites
  * Verify the submenu is not shown on Simple Sites
